RunningOrder/en.lproj/Localizable.strings

```diff
@@ -0,0 +1,24 @@
+/* 
+  Localizable.strings
+  RunningOrder
+
+  Created by Clément Nonn on 07/07/2020.
+  Copyright © 2020 Worldline. All rights reserved.
+*/
+
+// SprintsView - Section Header
```

Comment:
these comments looks like localised string comments ;)

Reply:
It is their purpose yes :P

Reply:
can we use them as such then? :)